source/components/__tests__/email.test.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-env jest */
+import {formatEmailParts} from '../send-email'
+
+describe('formatEmailParts', () => {
+	it('should format empty inputs', () => {
+		expect(formatEmailParts({to: ['']})).toBe('mailto:?body=&subject=')
+	})
+
+	it('should encode the "to" addressees', () => {
+		expect(formatEmailParts({to: ['test@domain.com']})).toBe(
+			'mailto:test%40domain.com?body=&subject=',
+		)
+		expect(formatEmailParts({to: ['test@domain.com', 'help@domain.com']})).toBe(
+			'mailto:test%40domain.com%2Chelp%40domain.com?body=&subject=',
+		)
+	})
+
+	it('should encode the "cc" addressees', () => {
+		expect(formatEmailParts({cc: ['test@domain.com']})).toBe(
+			'mailto:?body=&cc=test%40domain.com&subject=',
+		)
+		expect(formatEmailParts({cc: ['test@domain.com', 'help@domain.com']})).toBe(
+			'mailto:?body=&cc=test%40domain.com&cc=help%40domain.com&subject=',
```

Comment:
This seems likely problematic. I think we will need to join (but not encode) before we hand it off to openURL?

Reply:
Could you explain a little more about what the issue is?

Reply:
I think you're questioning what happens when we do the following 

```ts
let encodedTo = encodeURIComponent(['test@domain.com', 'help@domain.com'].join(','))
let query = querystring.stringify({cc, bcc, subject, body})

return `mailto:${encodedTo}?${query}`
```

specifically the encodedTo bit

```ts
 encodeURIComponent(['test@domain.com', 'help@domain.com'].join(','))
 ```
 
 which looks great after it is handed off to `openUrl`
 
![telegram-cloud-photo-size-1-5542185941918132147-y](https://user-images.githubusercontent.com/5240843/189504454-7ee343f5-f05e-4442-806c-fc53cd904247.jpg)

Reply:
I think it works well for "To:" - but we aren't doing the .join for "CC:" and "BCC:". We're adding two `&cc=` parameters, instead.

```
mailto:me@me.me?cc=a@b.com&cc=b@c.com
``` 

I just want to know what happens with that. 

I would expect something more like this:

```
mailto:me@me.me?cc=a@b.com,b@c.com
```

Reply:
Oh that makes sense. I will fix this.

Reply:
Okay. So oddly enough the implementation as-is does work for cc and bcc addresses. I do think it should be changed to be like your example, though.